dependencies.gradle

```diff
@@ -16,6 +18,9 @@ versions.springSecuritySamlVersion = "1.0.10.RELEASE" // this lib has reached EO
 versions.tomcatVersion = "9.0.75"
 versions.statsdClientVersion = "3.1.0" // statsd is not included in spring-boot 2.2 and later, so we need to specify the dependency explicitly
 versions.junitVersion = "4.13.2" // Needed to support junit 4 with spring-boot 2.4
+versions.snakeyamlVersion = "1.33" // Needed to resolve CVEs in internal spring-boot 2.7.12 inclusion of snakeyaml
```

Comment:
Why does this (plus the usage of this var below) need to defined in addition to `ext["snakeyaml.version"] = "1.33"`. I've tried to remove this var & did not notice difference in our final dependencies (as shown in intellij) after removing.

Reply:
Other than ^, LGTM

Reply:
It is redundant in the this cve only set of 4 libraries. I'll remove it.

Reply:
I've tried removing this var `versions.snakeyamlVersion = "1.33"` and its usage (aka changing `libraries.snakeyaml = "org.yaml:snakeyaml:${versions.snakeyamlVersion}"` back to libraries.snakeyaml = "org.yaml:snakeyaml" which is the existing code) and it seems to be work. Did you experiment with the same thing?

Reply:
I've tried removing this var `versions.snakeyamlVersion = "1.33"` and its usage (aka changing `libraries.snakeyaml = "org.yaml:snakeyaml:${versions.snakeyamlVersion}"` back to `libraries.snakeyaml = "org.yaml:snakeyaml"` which is the existing code) and it seems to be work (with `./gradlew build -x test`). Did you experiment with the same thing?

Reply:
The redundant snakeyaml versioning has been removed.